keyboards/ymd96/keymaps/AnthonyWharton/keymap.c

```diff
@@ -0,0 +1,48 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+Modified 2018 Anthony Wharton <th3ant@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define _DEFLT 0
+#define _RAISE 1
+
+#define KEYMAP LAYOUT_iso
```

Comment:
Why not replace the `KEYMAP`s with `LAYOUT_iso`?

Reply:
This is essentially why, a mixture of laziness/being a sheep and following the form of the other keyboard layouts in this `ymd96` folder. As you may or may not tell I'm new to QMK so I'm just following the immediate crowd until I fit in so to speak! :stuck_out_tongue_winking_eye: 

Perhaps if there is a strong opinion for one method of the other, this should be made a bit more explicit in the docs as if I recall correctly I didn't see anything explicitly on this?

I'll change it over for now as you brought it up, but happy for further discussion :smiley:

Reply:
Ah, okay.    Well, if you want/need this in there still, then it would be best in the keyboard's h file, after the LAYOUT_iso define.  

But in generally, we're trying to move away from "KEYMAP" completely.